package/upgrade/upgrade_node.sh

```diff
@@ -360,19 +380,31 @@ remove_rke2_canal_config() {
 
 convert_nodenetwork_to_vlanconfig() {
```

Comment:
@tjjh89017 @yaocw2020 

Please check the correctness of the change against `convert_nodenetwork_to_vlanconfig`, thanks.

The background is descripted in: https://github.com/harvester/harvester/issues/3098